site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        jcr:primaryType="cq:PageContent"
+        prefixPath="/content/aem-site-template-standard.theme/_default">
+        <items
+            jcr:primaryType="nt:unstructured">
+            <css
+                jcr:primaryType="nt:unstructured"
+                element="link"
+                location="header">
+                <attributes
+                    jcr:primaryType="nt:unstructured">
+                    <href
+                        jcr:primaryType="nt:unstructured"
+                        name="href"
+                        value="/css/theme.css"/>
+                    <rel
+                        jcr:primaryType="nt:unstructured"
+                        name="rel"
+                        value="stylesheet"/>                    
```

Comment:
Shouldn't it be: "preload stylesheet"?

Reply:
addressed in https://github.com/adobe/aem-site-template-standard/pull/71/commits/2e980e1ff80adef838ee0e4a4c7a40956bf40c55